src/read.c

```diff
@@ -3389,6 +3354,63 @@ static avifResult avifDecoderFlush(avifDecoder * decoder)
     return AVIF_RESULT_OK;
 }
 
+// Searches for the primary color item or the auxiliary alpha item.
+// Returns the target item if found, or NULL.
+static avifDecoderItem * avifDecoderFindItem(avifDecoder * decoder, avifBool alpha, uint32_t parentItemId)
```

Comment:
Strictly speaking this is separate from the introduction of avifTileGroup. Adding this on its own would have made this change more digestible.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1298.